be/src/exprs/expr_value.h

```diff
@@ -177,7 +177,7 @@ struct ExprValue {
             return &bigint_val;
 
         case TYPE_LARGEINT:
-            large_int_val = 0;
+            large_int_val = std::numeric_limits<int128_t>::min();
```

Comment:
I have a question.
Does std::numeric_limits supports int128_t?

Reply:
Yes. It does.